server/core.ts

```
@@ -159,6 +492,202 @@ export class WebSocketCore {
+        if (!id) {
+          this.sendError(ws, 'Missing player ID for shoot');
+          return;
+        }
+        // Broadcast shooting event to all other players
+        const shootData = {
+          id,
+          laserStart: restData.laserStart,
+          laserDirection: restData.laserDirection,
+          timestamp: Date.now()
+        };
+        this.broadcastToAll(
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)